File: ios/OrientationManager.m
//
//  OrientationManager.m
//  JawaAudit
//
//  Created by Purushotam Baheti on 18/04/25.
//

#import <Foundation/Foundation.h>
#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(OrientationManager, NSObject)

RCT_EXTERN_METHOD(lockToLandscape)
RCT_EXTERN_METHOD(lockToPortrait)

@end

File: ios/OrientationManager.swift
//
//  OrientationManager.swift
//  JawaAudit
//
//  Created by Purushotam Baheti on 18/04/25.
//

import Foundation

@objc(OrientationManager)
class OrientationManager: NSObject {
  @objc func lockToLandscape() {
    OrientationLocker.lockOrientation(.landscape)
  }

  @objc func lockToPortrait() {
    OrientationLocker.lockOrientation(.portrait)
  }
}
